dask_jobqueue/tests/test_pbs.py

```diff
@@ -5,11 +5,11 @@
 
 from dask.distributed import Client
 from distributed.utils_test import loop  # noqa: F401
-from pangeo import PBSCluster
+from dask_jobqueue import PBSCluster
 
 
 def test_basic(loop):
-    with PBSCluster(walltime='00:02:00', threads_per_worker=2, memory='7GB',
+    with PBSCluster(walltime='00:02:00', threads=2, memory='7GB',
```

Comment:
I'm curious if this name is perhaps a little ambiguous now.  It might be worth it for the shorter name, I'm not sure.

Reply:
If it would be me, I would name every corresponding attribute with suffix liker "per_worker" or prefix like "worker_". Like mentioned in https://github.com/dask/dask-jobqueue/issues/7#issuecomment-370579557.
As it stands, I don't see why threads should be treated differently than memory. Or maybe we should change memory to memory_per_worker, as for processes it actually translates in "number of workers"?